docs/modding/4-spriting.md

```diff
@@ -1,3 +1,90 @@
 # Spriting
 
-//TODO
+### Spriting is an essential part of mindustry modding, without it everything you've made in JSON, HJSON, or Script [JS] won't be visible
+It is highly recomended that you have an image editing software that supports transparency.
+
+This includes but not limited to: 
+  - Aseprite 
+  - Piskel
+  - GIMP
+  - Paint.net
+
+## Size
+The smallest block size you could make is `32×32` size, which is a 1×1 block, making bigger block means increasing the sprite size by additional `32`, so a 2×2 block is `64×64` and so on, this applies to both turrets and blocks.
+- `1×1` : `32×32`
+- `2×2` : `64×64`
+- `3×3` : `96×96`
+- `4×4` : `128×128`
+- `5×5` : `160×160`
+
+However you are not limited to those options, the game will still load sprites bigger or smaller than the block you've assigned it for, which can result in a unique looking sprite or an atrocity.
+
+## Storing Sprites
+Sprites can simply be dropped in the `sprites/` subdirectory. The content parser will look through it recursively, so you can organize them how ever you feel. 
```

Comment:
This isn't true, see https://github.com/MindustryGame/wiki/blob/master/docs/modding/1-modding.md#sprites

Reply:
Which part of it isn't true?
The game does load for example 64*64 sprite for a 1*1 block, it just looks off. The sprites will cover a larger area but it will still occupy 1*1 tile

Reply:
"The first directory in sprites/, e.g. sprites/blocks, determines the page in this atlas that sprites are put in. Putting a block's sprite in the units page is likely to cause lots of lag"

Reply:
_It's still an option, just ridiculously inefficient._

Reply:
ah okay okay, ill change it